src/allotropy/parsers/beckman_vi_cell_xr/vi_cell_xr_parser.py

```diff
@@ -144,7 +143,9 @@ def _get_cell_counting_document_item(
                                     ),
                                     viability__cell_counter_=viability__cell_counter_,
                                     viable_cell_density__cell_counter_=viable_cell_density__cell_counter_,
-                                    total_cell_count=total_cell_count,
+                                    total_cell_count=get_property_from_sample(
+                                        sample, "Total cells"
```

Comment:
Could you create a constant for "Total cells" and use that here and in `property_lookup`?

Reply:
I'll do it for all properties in that lookup dict

Reply:
better idea: #269

Reply:
@bdworth I think is a good idea but also think this should be blocking. Can we merge this PR and then proceed with the other changes?

Reply:
#269 has merged. Sorry, I only saw this comment now.

Reply:
I should be able to approve quickly once you fix conflicts!